app/invocation/invocation.tsx

```diff
@@ -518,7 +518,7 @@ export default class InvocationComponent extends React.Component<Props, State> {
         {!isBazelInvocation && (
           <div className="container">
             <div className="workflow-details-header">
-              <h2>Runner results (advanced)</h2>
+              <h2>Runner results</h2>
```

Comment:
sorry, I updated this part here: https://github.com/buildbuddy-io/buildbuddy/pull/7428

Reply:
Oh no worries - missed that!